docs/guides/tutorials/getting_started/connect_to_data.rst

```diff
@@ -14,88 +14,71 @@ Let's configure your first Datasource, by following the next steps in the CLI in
 
-Datasources solve this problem, by conceptually separating *what* you want to validate from *how* you want to validate it. Datasources give you full control over the process of bringing data and Expectations together, then abstract away that underlying complexity when you validate X data against Y Expectations.
+    ...
 
-.. figure:: ../../images/datasource-conceptual-diagram.png
-    :width: 400px
-    :class: with-shadow float-right
+    Would you like to profile new Expectations for a single data asset within your new Datasource? [Y/n]: n
 
-We call the layer that handles the actual computation an :ref:`Execution Engine <reference__core_concepts__validation__execution_engines>`. Currently, Great Expectations supports three Execution Engines: pandas, sqlalchemy, and pyspark. We plan to extend the library to support others in the future.
+That's it! You just configured your first Datasource! Make sure to choose ``n`` at this prompt to exit the ``init`` flow for now.
 
-The layer that handles connecting to data is called a :ref:`Batch Kwargs Generator <reference__core_concepts__batch_kwargs_generators>`. Not all Batch Kwarg Generators can be used with all Execution Engines. It's also possible to configure a Datasource without a Batch Kwargs Generator if you want to pass data in directly. However, Batch Kwarg Generators are required to get the most out of Great Expectations' :ref:`Data Docs` and :ref:`Profilers`.
-
-You can read more about the inner workings of Datasources, Execution Engines, and Batch Kwargs Generators :ref:`here <Validation>`.
-
-.. attention::
-
-    We plan to upgrade this configuration API with better names and more conceptual clarity prior to Great Expectations' 1.0 release. If at all possible, we will make those changes in a non-breaking way. If you have ideas, concerns or questions about this planned improvement, please join the public discussion `here <https://discuss.greatexpectations.io/t/conceptual-mismatches-in-datasource-internals/134>`__.
+**Before continuing, let's stop and unpack what just happened.**
 
 
 Configuring Datasources
 -----------------------
 
-When you completed those last few steps in ``great_expectations init``, you told Great Expectations that
+When you completed those last few steps in ``great_expectations init``, you told Great Expectations that:
 
-1. You want to create a new Datasource called ``my_data__dir``.
-2. You want to use Pandas as your :ref:`Execution Engine <Execution Engines>`, hence ``data_asset_type.class_name = PandasDataset``.
-3. You want to create a BatchKwarg Generator called ``subdir_reader`` using the class ``SubdirReaderBatchKwargsGenerator``.
-4. This particular Generator connects to data in files within a local directory, specified here as ``../my_data``.
+1. You want to create a new Datasource called ``my_postgres_db``.
+2. You want to use Postgres as your :ref:`Execution Engine <Execution Engines>`, hence ``data_asset_type.class_name = SqlAlchemyDataset``.
```

Comment:
I can't actually find a single reference to Execution Engine in the docs. I left the ref as a placeholder - which page should this link to?

Reply:
I think you're right -- it's not there.

I think I'd reword this as follows:
```suggestion
2. You want to use SqlAlchemy to evaluate your Expectations, hence ``data_asset_type.class_name = SqlAlchemyDataset``. Your database configuration tells the SqlAlchemyDataset to use a Postgresql-specific driver.
```
The term execution engine *does* appear in the draft updated core concepts doc, but it's better to let all those names settle before surprising users with this one out of the blue.

Reply:
:+1: